app/models/notebook.rb

```diff
@@ -31,8 +30,6 @@ class Notebook < ActiveRecord::Base
   validates :uuid, uniqueness: { case_sensitive: false }
   validates :uuid, uuid: true
 
-  after_destroy :remove_content, :remove_wordcloud
```

Comment:
Still want the `after_destroy :remove_content` here to remove the ipynb when the database entry goes away.

Reply:
OOops...overzealous there...will fix that right now